src/components/split-pane/SplitPane.tsx

```diff
@@ -99,7 +99,6 @@ export function SplitPane(props: SplitPaneProps) {
 
   const splitterRef = useRef<HTMLDivElement>(null);
   const { onMouseDown } = useSplitPaneSize({
-    mainSide,
```

Comment:
I think the hook should still depend on this somehow. It doesn't work now when the value is `'end'`:
https://deploy-preview-228--analysis-ui-components.netlify.app/stories/?arg-r-mainSide=end&story=components--splitpane--control
https://deploy-preview-228--analysis-ui-components.netlify.app/

Reply:
fixed thank you